### PR TITLE
[ADD][14.0] purchase_order_shipping_date

### PR DESCRIPTION
**Pull request by AshishHirapara**
initial commit: adds a Purchase Shipping Date on the PO report

### CODE REVIEW

File: purchase_order_shipping_date/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Order Shipping Date",
+    "version": "14.0.1.0.0",
+    "author": "Ashish Hirpara, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Purchase",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "summary": """Display Purchase Order Shipping Date in PO Report""",
```
Comment (francesco-ooops) [inline purchase_order_shipping_date/__manifest__.py]:
```suggestion
    "summary": """Add field Shipping Date in PO and Report""",
```

File: purchase_order_shipping_date/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This module adds a field 'Shipping date' in purchase order, it will be displayed in PO report.
```
Comment (francesco-ooops) [inline purchase_order_shipping_date/readme/DESCRIPTION.rst]: This field can be used to mark expected shipping date by vendor.

**Review by francesco-ooops — APPROVED** [review]
Functional review ok, this module arises from the need to mark shipping date on PO (standard odoo only allows to set reception date)

File: purchase_order_shipping_date/__manifest__.py (hunk 1 above)
Comment (AshishHirapara) [reply purchase_order_shipping_date/__manifest__.py]: done

File: purchase_order_shipping_date/readme/DESCRIPTION.rst (hunk 2 above)
Comment (AshishHirapara) [reply purchase_order_shipping_date/readme/DESCRIPTION.rst]: updated

**Review by francesco-ooops — APPROVED** [review]

**Review by SicurSam — APPROVED** [review]
Functional review ok!

**Comment by francesco-ooops** [comment]
@OCA/procurement-purchase-maintainers what do you think?

**Review by etobella — APPROVED** [review]
LGTM

**Pull request merged**